srcpkgs/capnproto/template

```diff
@@ -10,15 +10,22 @@ maintainer="eater <hello@eaterofco.de>"
 license="MIT"
 homepage="https://capnproto.org"
 distfiles="${homepage}/capnproto-c++-${version}.tar.gz"
-checksum=d1f40e47574c65700f0ec98bf66729378efabe3c72bc0cda795037498541c10d
-
-CXXFLAGS="-std=c++14"
+checksum=83680aaef8c192b884e38eab418b8482d321af6ae7ab7befa3a9370b8e716aad
 
 if [ "$CROSS_BUILD" ]; then
 	hostmakedepends+=" capnproto-devel"
 	configure_args+=" --with-external-capnp"
 fi
 
+if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
+	makedepends+=" libatomic-devel"
+	LDFLAGS+=" -latomic"
+fi
+
+if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
+	CPPFLAGS+=" -DKJ_USE_FIBERS=0"
```

Comment:
You should be able to just set these (`=`) instead of adding to them.

Reply:
Done